### DISCUSSION TITLE
Models, Checkpoints help!

### DISCUSSION THREAD

**Discussion by Camera-Shake** [lllyasviel/stable-diffusion-webui-forge#267] [General]
Hi I was hoping someone can help me out. I have just installed the Forge webui and starts okay, except this is my fresh install and was wondering how and where you can download from all required models, Loras, Extensions, Checkpoints??? Is there a quicker way I can batch download these and put them in their right folders? I have to start with the latest SD XL turbo .... what kind of workflow do you suggest considering I never had Automatic1111 or SD installed before?

**Comment by huchenlei** [comment]
You can download base model and LoRA from https://civitai.com/

See https://github.com/lllyasviel/stable-diffusion-webui-forge/wiki/ControlNet-model-download for controlnet model download.
